src/tcgwars/logic/impl/gen4/SecretWonders.groovy

```diff
@@ -2679,17 +2679,26 @@ public enum SecretWonders implements LogicCardInfo {
         return basicTrainer (this) {
           text "You can play only one Supporter card each turn. When you play this card, put it next to your Active Pokémon. When your turn ends, discard this card.\nChoose a card from your hand and put it on top of your deck. Search your deck for a Pokémon, show it to your opponent, and put it into your hand. Shuffle your deck afterward. (If this is the only card in your hand, you can’t play this card.)"
           onPlay {
+            my.hand.getExcludedList(thisCard).select("Choose the card to put back in your deck").showToOpponent("Chosen card")
+            my.deck.search(count:1,"Choose the Pokémon to put back in your deck",cardTypeFilter(POKEMON)).showToOpponent("Chosen card")
           }
           playRequirement{
+            assert my.hand.size() > 1 : "You can't play this card"
```

Comment:
Could you change this message to say `Your hand needs one other card to play this card` or something along those lines?